src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -22,8 +22,18 @@
                            'Exception will be thrown if free instances exceed one.')
 register_cli_argument('iot hub create', 'unit', help='Units in your IoT Hub.', type=int)
 
+# Arguments for 'iot hub show-connection-string'
+register_cli_argument('iot hub show-connection-string', 'name', options_list=('--name', '-n'),
+                      help='Target IoT Hub name.')
```

Comment:
Good suggestion. However I am afraid I don't have enough time to add the completer in this PR.
Actually we are in a hurry to get this PR merged. Because we will do a review with our partners at this Friday. And these two commands are really important in the experience we want to demo. I know the tight schedule sounds like crazy. But we are kind of in a stage which things are changed rapidly and we have no choice but doing whatever needs to be done.
So I would love this PR to be merged today. And in the next PR (at next week), I will fix all missing parts here.
Thanks for your understanding! 😄